plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -94,6 +94,7 @@ open class KtlintPlugin : Plugin<Project> {
         // Add the args to enable verbose and debug mode.
         if (extension.verbose) runArgs.add("--verbose")
         if (extension.debug) runArgs.add("--debug")
+        if (extension.android) runArgs.add("--android")
```

Comment:
Can you add a check for ktlint version before applying `--android` as it was only introduced since `0.12.0` and current project default is `0.9.2`

Reply:
Sure.